### ISSUE TITLE
Connection to RouterDC (NeurIPS 2024)

### ISSUE THREAD

**Issue by shuhao02** [lmarena/p2l#4]
Hi Prompt-to-Leaderboard Team/Community,

Thank you for the outstanding work! P2L produces leaderboards specific to a prompt in Chatbot Arena, which greatly shows the effectiveness of the router-based method.

I’m excited to share our recent work in **NeurIPS 2024**, RouterDC, which we believe aligns closely with the goals and challenges addressed in Prompt-to-Leaderboard. RouterDC is a novel framework to learn a router to select the suitable LLM for each query by dual contrastive learning.

Key Features of RouterDC:
- RouterDC learns a router to select the suitable LLM for each query by dual contrastive learning, which consists of sample-LLM and sample-sample contrastive losses
- RouterDC is **parameter-efficient** (has less than 100M parameters) and **computation-efficient** (without backpropagating the gradients through LLMs) in training.
- Results on various datasets (including MMLU, GSM8K, CMMLU, ARC-C, HumanEval) show the effectiveness of RouterDC, outperforming individual top-performing LLMs as well as existing routing methods on both in-distribution (+2.76%) and out-of-distribution (+1.90%) tasks.

Github: https://github.com/shuhao02/RouterDC
Paper: https://arxiv.org/abs/2409.19886

**Comment by DoiiarX** [comment]
I feel that your project introduction is not detailed enough...

**Comment by shuhao02** [comment]
Thank you for your valuable suggestion! We've added an abstract to the README file to provide a clear introduction to our project. If you have any further ideas or feedback, please feel free to open an issue in our project repository.

**Comment by efrick2002** [comment]
Thanks for sharing!
